systemtest/src/test/java/io/strimzi/systemtest/specific/SpecificIsolatedST.java

```diff
@@ -334,7 +342,14 @@ void testRackAwareConnectCorrectDeployment(ExtensionContext extensionContext) {
         assertThat(connectPodNodeSelectorRequirement.getKey(), is(rackKey));
         assertThat(connectPodNodeSelectorRequirement.getOperator(), is("Exists"));
 
-        KafkaConnectUtils.sendReceiveMessagesThroughConnect(connectPodName, topicName, kafkaClientsPodName, clusterOperator.getDeploymentNamespace(), clusterName);
+        // fetch scraper Pod name
+        final LabelSelector scraperLabelSelector = new LabelSelectorBuilder()
+            .addToMatchLabels(Constants.SCRAPER_LABEL_KEY, Constants.SCRAPER_LABEL_VALUE)
+            .build();
+        final String scraperPodName = kubeClient(clusterOperator.getDeploymentNamespace())
+            .listPods(clusterOperator.getDeploymentNamespace(), scraperLabelSelector).get(0).getMetadata().getName();
```

Comment:
don't we have a method for that? if not, shouldn't we have one? I think there will be more occurrences of this.

Reply:
We don't but I added it. )